### ISSUE TITLE
Lexer: Leading dot doesn’t close function passed as argument and not wrapped in parentheses

### ISSUE THREAD

**Issue by bwiklund** [jashkenas/coffeescript#3446]
On 1.7.1, this breaks:

``` coffeescript
angular.module 'foo'
.controller 'EmailLoginCtrl', ->
.controller 'EmailSignupCtrl', ->
```

While this is fine:

``` coffeescript
angular.module 'foo'
.controller 'EmailLoginCtrl', (->)
.controller 'EmailSignupCtrl', (->)
```

And so is this:

``` coffeescript
angular.module 'foo'
.controller 'EmailLoginCtrl', -> return
.controller 'EmailSignupCtrl', -> return
```

Is this the way it's supposed to work?

**Comment by GeoffreyBooth** [comment]
You’re really trying to spread across multiple lines what is really a one-line call. When you do so, you should use `\`, and parentheses as necessary:

```coffee
angular.module('foo')\
.controller('EmailLoginCtrl', ->)\
.controller('EmailSignupCtrl', ->)
```

**Comment by lydell** [comment]
I still think this is a bug.

**Comment by GeoffreyBooth** [comment]
Fixed by #4588.
